Rakefile

```diff
@@ -54,6 +54,27 @@ namespace :spec do
   desc "Run all javascript specs"
   task :javascript => ["app:test:initialize", :environment, "jasmine:ci"]
 
+  desc "Run all cypress specs"
+  task :cypress => ["app:cypress:ui:run"]
+
+  namespace :cypress do
```

Comment:
For those who might re-reviewing this again, this section was mostly to running get things running via `manageiq-cross_repo-tests` PR I have been using:

https://github.com/ManageIQ/manageiq-cross_repo-tests/pull/150

I was trying to write too much bash scripting inside of environment variables, so this was a much easier way to not only debug, but also my idea standardize things a bit using `rake`.

That said, this probably won't be needed if we are running things from this repo, and I expect this is probably not a preferred way of doing things by most, so we can remove this once the core changes have been merged.  I can always archive this commit somewhere to consider as an option in the future.